### PR TITLE
Interactive To-Do-List

### PR DESCRIPTION
**Pull request by cvalencia1991**
# In this milestone, I did the following activities:

- Interact add and remove Buttons
- add in the Webpack to import images correctly
- Create in the local storage the info in an array 
- Edit files with linters

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,26 +3,23 @@ import './imgs/enter.png';
 import './imgs/menu.png';
```
Comment (Lucas-Masaba) [inline src/index.js:4]: - [ ] Please add a new javascript file for the new functionality which is to add and remove tasks as required in the instructions 👍 .
- [ ] When I add a task and refresh the page, it returns an undefined task. Please fix that.
image
- [ ] When I delete an item and refresh the page, it just returns the item as checked and remains on the local storage.

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @cvalencia1991  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (cvalencia1991) [reply src/index.js:4]: sorry but im stuck right there i don't know how to do it can you help me

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @cvalencia1991, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

work4

 ## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️

 ## Required Changes ♻️

Kindly test your system and it can't delete a specific task and it can't delete completed tasks either, I suggest reviewing your scripts in those areas and making any necessary adjustments before submitting a PR again

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**